src/main/kotlin/io/sip3/salto/ce/sdp/SdpHandler.kt

```diff
@@ -73,8 +73,17 @@ class SdpHandler : AbstractVerticle() {
     private fun handle(transaction: SipTransaction) {
         val session = SdpSessionDescription().apply {
             callId = transaction.callId
-            request = transaction.request?.sessionDescription()?.getMediaDescription("audio") ?: return
-            response = transaction.response?.sessionDescription()?.getMediaDescription("audio") ?: return
+            try {
+                request = transaction.request?.sessionDescription()?.getMediaDescription("audio") ?: return
+            } catch (e: Exception) {
+                logger.debug(e) { "request.sessionDescription() failed. CallId: $callId" }
```

Comment:
Let's better use one and the same message: `Couldn't parse SDP. Message: ${transaction.request}`